ques9.cpp

```diff
@@ -0,0 +1,40 @@
+Time Complexity-O(log n)
+Space Complexity-O(n)
+Did the code execute successfully on Leetcode-Yes
```

Comment:
The space complexity is O 1

Reply:
I'm sorry i will update that.
It was by mistake.

Thanks for the update.